airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -37,10 +35,14 @@ def __init__(self, kube_config):
 
         self.dags_volume_name = 'airflow-dags'
         self.logs_volume_name = 'airflow-logs'
+        self.git_sync_ssh_secret_volume_name = 'git-sync-ssh-key'
+        self.git_ssh_key_secret_key = 'gitSshKey'
+        self.git_sync_ssh_known_hosts_volume_name = 'git-sync-known-hosts'
+        self.git_ssh_known_hosts_configmap_key = 'known_hosts'
```

Comment:
Oh, since these are hard-coded lets move them to "class properties":

```python
class WorkerConfiguration(LoggingMixin):
    """Contains Kubernetes Airflow Worker configuration logic"""

    dags_volume_name = 'airflow-dags'
    logs_volume_name = 'airflow-logs'
    git_sync_ssh_secret_volume_name = 'git-sync-ssh-key'
    git_ssh_key_secret_key = 'gitSshKey'
    git_sync_ssh_known_hosts_volume_name = 'git-sync-known-hosts'
    git_ssh_known_hosts_configmap_key = 'known_hosts'

    def __init__(self, kube_config):
```

Nothing else needs to change about how these are used.